.storybook/Wrapper.tsx

```diff
@@ -10,11 +10,19 @@ const defaultTheme = 'baseTheme';
 export default (storyFn) => {
 	const [theme, setTheme] = useState(themes.get(defaultTheme));
 
-	const selectedTheme = select('Theme', themes.keys(), defaultTheme);
+	const selectedTheme = select(
+		'Theme',
+		Array.from(themes.keys()),
+		defaultTheme,
+	);
 
 	useEffect(() => {
 		setTheme(themes.get(selectedTheme));
 	}, [selectedTheme]);
 
-	return <Frame theme={theme}>{storyFn()}</Frame>;
+	return (
+		<Frame theme={theme}>
+			<StrictMode>{storyFn()}</StrictMode>
```

Comment:
Did this so we get warnings during development when we're doing not-so-right things. ✅